WordPress/Classes/ViewRelated/Blog/QuickStartTours.swift

```diff
@@ -7,6 +7,49 @@ protocol QuickStartTour {
     var icon: UIImage { get }
 }
 
+fileprivate enum Constants {
+    static let iconOffset: CGFloat = -1.0
+    static let highlightColor = WPStyleGuide.lightBlue()
+}
+
+
+extension QuickStartTour {
+    func createHighlightMessage(base normalString: String, highlight: String, icon: UIImage) -> NSAttributedString {
```

Comment:
Nitpick, but the Swift API Guidelines recommend `make` instead of `create`

Reply:
Ah, yes, you're totally correct!